doc/user_guide/io.rst

```diff
@@ -1361,8 +1361,10 @@ some additional loading arguments are provided.
 Extra loading arguments
 +++++++++++++++++++++++
 
-- ``dataset_keys``: ``None``, ``str`` or ``list`` of strings - Default is ``None`` . Absolute path(s) or string(s) to search for in the path to find one or more datasets.
+- ``dataset_keys``: ``None``, ``str`` or ``list`` of strings - Default is ``None`` . String(s) to search for in the path to find one or more datasets.
+- ``dataset_paths``: ``None``, ``str`` or ``list`` of strings - Default is ``None`` . Absolute path(s) to search for in the path to find one or more datasets.
```

Comment:
There is already a `dataset_path` used for other hdf5 file reader. Would it be worth using the same without `s`? The drawback is that this is not consistent with `dataset_keys` or `metadata_keys`.
Maybe, it would be good to define if we stick to plural or singular in this case... and deprecate/rename arguments accordingly. I don't know if one is better than the other!

Reply:
Personally I would prefer to go with the plural as it implies that multiple paths can be searched for, but happy to go with singular for consistency with other readers if it is more suitable.

Reply:
Actually, after looking at the use of singular versus plural in argument name, it seems that `numpy` (and `hyperspy`) uses the `axis` argument (see for example `sum`), which can take one or several items, so I would suggest to use the singular.

Reply:
I deprecated the plural arguments and now use singular for nexus file io.